sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -19,8 +19,8 @@
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Scale;
 using Microsoft.Extensions.Logging;
-using Microsoft.Azure.Amqp.Framing;
-using Newtonsoft.Json.Schema;
+using static System.Collections.Specialized.BitVector32;
```

Comment:
Ok I'm pretty sure you are messing with us now 😆

Reply:
😨 I think I need to just turn off the vs automatic using statements